src/unity/python/turicreate/CMakeLists.txt

```diff
@@ -49,6 +49,15 @@ set_property(DIRECTORY PROPERTY INSTALLATION_EXTENSIONS "${INSTALLATION_EXTENSIO
 set_property(DIRECTORY PROPERTY INSTALLATION_BINARY_TARGETS "${INSTALLATION_BINARY_TARGETS}")
 set_property(DIRECTORY PROPERTY INSTALLATION_BINARY_FILES "${INSTALLATION_BINARY_FILES}")
 
+add_custom_target(
+  visualization_vega ALL
+  DEPENDS visualization_client
```

Comment:
Does this actually depend on visualization_client? If we are committing the files being copied directly into the repo, it seems like it doesn't.